test/api/test_tools.py

```diff
@@ -1867,6 +1867,23 @@ def test_map_over_collection_structured_like(self):
             assert second_collection_level['collection_type'] == 'list'
             assert second_collection_level['elements'][0]['element_type'] == 'hda'
 
+    @skip_without_tool("__SPLIT_COLLECTION_BY_NUMBER__")
+    def test_split_collection_by_number(self):
+        with self.dataset_populator.test_history() as history_id:
+            hdca_id = self.dataset_collection_populator.create_list_in_history(history_id, contents=[("A", "A"), ("B", "B")]).json()['id']
+            self.dataset_populator.wait_for_history(history_id, assert_ok=True)
+            inputs = {
+                "input": {'values': [dict(src="hdca", id=hdca_id)]},
```

Comment:
providing this as a list of dicts is meant for when you run multiple inputs in batch, you can use
```suggestion
                "input": {'values': {'src': 'hdca', 'id': hdca_id}},
```
here.